Shell/Options.cpp

```diff
@@ -179,6 +181,11 @@ void Options::init()
     _lookup.insert(&_slowness);
     _slowness.onlyUsefulWith(UsingPortfolioTechnology());
 
+    _randomizedPortfolioWorkers = BoolOptionValue("randomized_portfolio_workers","",true);
```

Comment:
Is this option name clear? From the option name, I assumed some extra randomization going on but that's not quite what's going on.

Out of interest... portfolio mode isn't really deterministic but I'm guessing setting this to true makes it 'less' deterministic (?)

Reply:
Happy to discuss a better name here! "randomize_seed_for_portfolio_workers"?

The main use case is this: we have strategy A, later, thanks to getExtraSchedules, we have strategy A+shuffling and later still we have it again (perhaps with higher limit). If the workers inherit a fixed seed from the master process running A+shuffling twice in a row will not be as effective as with different seeds. 

But let's also discuss, if it should be on by default. It though it's fine, as long as proper competition modes have it turned off. Also, without explicit shuffling there a little use of the random bits by vampire. If I remember correctly, it's in the skiplists of the passive clause containers (only influences timing, not decisions) and in global_subsumption_explicit_minim which is actually "randomized" by default.